dotnet/src/SemanticKernel.Abstractions/AI/TextToImage/ImageContent.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.SemanticKernel.TextToImage;
```

Comment:
Out of scope of this PR, but I think would be useful to move all the content abstractions to the root namespace.

- ContentBase
- ChatMessageContent
- TextContent
- ImageContent

Streaming also.

```suggestion
namespace Microsoft.SemanticKernel;
```

Reply:
Thanks, I think it's a good idea. In case we make a decision to do that, I will create a separate issue to avoid big number of changes in this PR.